pkg/kubectl/cmd/util/factory_client_access.go

```diff
@@ -454,8 +454,8 @@ const (
 	SecretForDockerRegistryV1GeneratorName      = "secret-for-docker-registry/v1"
 	SecretForTLSV1GeneratorName                 = "secret-for-tls/v1"
 	ConfigMapV1GeneratorName                    = "configmap/v1"
-	ClusterRoleBindingV1GeneratorName           = "clusterrolebinding.rbac.authorization.k8s.io/v1alpha1"
```

Comment:
huh, that's neat.  @derekwaynecarr you had a vision of stable output for generators, but the generators are all building internal objects which will serialize out differently depending on how execute them.  Were you thinking that generator versions would be divorced from the API versions themselves.

Also, @xilabao given some people's views on generators these have to remain under these names and might even have to continue producing now incorrect, but stable output.